ecommerce/backend/API/model_serializers.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework import serializers
+from .models import User
+from .models import Product
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = '__all__'
+
+class ProductSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Product
+        fields = '__all__'
```

Comment:
Can you place your serializer file under serializers folder and import it in __init__.py file located in serializers folder.
Functionally it is okay but to keep file structure organized, it would be better. You can have a product_serializers.py under serializers folder. 
Adding 
from .product_serializer import *
to init py in serializers folder will handle it.
After that any time you want to use your serializer you can import it like
from API.serializers import product_serializers